webapp/src/components/Header.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useEffect, useMemo } from 'react'
+import React, { useEffect, useMemo, useState } from 'react'
 import Grid2 from '@mui/material/Grid2'
 import { useSelector, useDispatch } from 'react-redux'
 import EnvironmentVars from '../EnvironmentVars'
```

Comment:
```suggestion
import EnvironmentVars from '../EnvironmentVars'
import Box from '@mui/material/Box'
```

Importing this so it can be used to further refine layout